pkgs/stdenv/generic/check-meta.nix

```diff
@@ -144,9 +144,9 @@ let
     ''
 
       Alternatively you can configure a predicate to allow specific packages:
-        { nixpkgs.config.${predicateConfigAttr} = pkg: builtins.elem (lib.getName pkg) [
+        { nixpkgs.config.${predicateConfigAttr} = pkg: builtins.elem (lib.getName pkg) (map lib.getName [
             "${lib.getName attrs}"
```

Comment:
I'm looking into how this should be changed to get the attribute name from nixpkgs rather than the package name.